sass/base/_components.scss

```diff
@@ -30,10 +30,10 @@
 @import "components/media-block/media-block";
 @import "components/media/media";
 @import "components/meta-icons/meta-icons";
+@import "components/navigation/feature-nav";
```

Comment:
Ahhh, right. I think it'll probably be okay (it's not set up to actually 'inherit' or 'include' in the css/sass way); thinking about it, I had @Heleen-cr use a more specific selector in the feature nav so the order might not actually matter.

I'll keep an eye on it, though.